server/broadcast.go

```diff
@@ -386,6 +386,19 @@ func processSegment(cxn *rtmpConnection, seg *stream.HLSSegment) ([]string, erro
 	if ros != nil {
 		go func() {
 			now := time.Now()
+			hasZeroVideoFrame, err := ffmpeg.HasZeroVideoFrameBytes(seg.Data)
```

Comment:
I see that this function was added in https://github.com/livepeer/lpms/pull/237 and it returns an integer. Fine for now since that PR was already merged, but I think returning a boolean would make this function more straightforward to use and suggest making that change separately.